seatunnel-translation/seatunnel-translation-spark/seatunnel-translation-spark-common/src/main/java/org/apache/seatunnel/translation/spark/common/serialization/InternalRowConverter.java

```diff
@@ -74,13 +73,12 @@ private static Object convert(Object field, SeaTunnelDataType<?> dataType) {
                 SeaTunnelRowType rowType = (SeaTunnelRowType) dataType;
                 return convert(seaTunnelRow, rowType);
             case DATE:
-                return Date.valueOf((LocalDate) field);
+                return (int) ((LocalDate) field).toEpochDay();
             case TIME:
-                //TODO: how reconvert?
-                LocalTime localTime = (LocalTime) field;
-                return Timestamp.valueOf(LocalDateTime.of(LocalDate.ofEpochDay(0), localTime));
+                // TODO: how reconvert? there are not Support for Spark Type,we must define the Type in spark
+                throw new RuntimeException("we not support for time type now");
```

Comment:
Please replace `we not support for time type now` with `we do not support for time type now`